pom.xml

```diff
@@ -31,9 +30,41 @@
     <nsxpolicysdk.version>3.1.5.0.0</nsxpolicysdk.version>
     <nsxvmcawsint.version>3.1.5.0.0</nsxvmcawsint.version>
     <nsxvmcsdkcommon.version>3.1.5.0.0</nsxvmcsdkcommon.version>
-    <vmc.draas.version>1.24.0</vmc.draas.version>
+    <vmc.draas.version>1.25.0</vmc.draas.version>
```

Comment:
Why is this so much ahead than the version on the python side?

Reply:
Due to some fixes in the java specific runtime and generators, number of releases in case of java were more than the python one.